docs/spicedb/commands.md

```diff
@@ -0,0 +1,213 @@
+# SpiceDB Commands
```

Comment:
I really don't find this doc useful. I like the sections but it should instead just be written descriptions of the intent of each command in case someone is confused by looking at the flags.

Reply:
Removed the page to revisit later